assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| PMC               | Partially | Scrum has the notion of sprint goals. This can be used to determine how well a team is performing based on how often they reach their sprint goals. There is also velocity which can be used to plan and determine if any form of attempted process improvement are adding any value. Scrum specifies that the Scrum master is responsible for for making sure that the team performs optimally but does not formally state everything they should do and how it should be done.           |
+| PP                | Fully     | This is corresponds with sprint planning meetings where backlog items are selected and broken down into tasks for the upcoming sprint backlog which are then assigned to team members.             |
+| PPQA              | Partially | Scrum prescribes some practices to measure how processes are performed and can for example be used to by the Scrum master to know when they should aid the team but it says nothing about how about to assure the quality of products produced using it. Most Scrum teams have to incorporate quality assurance into their definition of done for sprint tasks            |
+| REQM              | Fully | The requirements are in user stories in the backlog and are the responsibility of the product owner to determine and formulate.            |
+| SAM               | Not at all | Scrum only prescribes some practices on how a team should develop software and does not have information on dealing with a supplier.             |
+| **Level 3**                                 |
+| DAR               | Not at all | Scrum does not have any formal evaluation method which can be used analyze possible decisions or alternatives            |
+| IPM               | Partially | There are guidelines in place to manage when stakeholders can interact with its team but for the most part this up to the product owner            |
+| OPD               | Not at all | This is left entirely up to the wishes of the development team            |
+| OPF               | Partially | Scrum's retrospective allows for teams to come up with improvements to the process.             |
+| OT                | Not at all | Identifying training needs of an organization and assessing its effectiveness is not covered in Scrum. |
+| PI                | Partially | Scrum states that at the end of each sprint a product should be developed, tested and ready for production. It does not cover how this integration should be achieved. |
+| RD                | Partially | Scrum addresses this through the use of user stories in the Product Backlog and Sprint Backlog. Its responsibility however is almost entirely delegated to the Product Owner and Scrum does not offer guidelines in how to collect and coördinate stakleholder needs and translate them in requirements.           |
```

Comment:
Its responsibility refers to putting the stories in the backlogs, which isn't mentioned explicitly. Therefore it is probably better to explictly mention this instead of using Its